### PR TITLE
Add a note to 'Running SSSD as a non-root user' design page.

### PR DESCRIPTION
**Pull request by alexey-tikhonov**

### CODE REVIEW

**Review by justin-stephenson — DISMISSED** [review]
Ack, thank you.

File: src/design-pages/not_root_sssd.rst [hunk 1]
```diff
@@ -23,6 +23,14 @@ This is a general use-case, following the principle of least privilege.
 The processes should not run as root unless they really need the root
 privileges.
 
+Important note
+--------------
+Text below describes initial design. sssd-2.10 release features significant
```
[the remark this answers is not in the document]
Comment (andreboscatto) [reply src/design-pages/not_root_sssd.rst:28]: I'm still not sure if we should start with "The initial design is described below." or move it to the end.

[the remark this answers is not in the document]
Comment (alexey-tikhonov) [reply src/design-pages/not_root_sssd.rst]: Updated.

**Review by andreboscatto — APPROVED** [review]
LGTM

**Pull request merged**